### PR TITLE
WIP Overfitting chapter refactoring

### PR DESCRIPTION
**Pull request by ogrisel**
I splitted the slides and updated the jupyter book structure with an intro and take away.

I still need to add the quizzes.

I also want to improve the diagrams of the bias vs variance slides to progressively add arrows to the target before recording the voice-over.

### CODE REVIEW

File: jupyter-book/overfit/overfit_intro.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Then we will present a methodology to quantify those problems by **contrasting the
+train error with the test error** for various choice of the model family,
+model parameters. More importantly, we will emphasize the **impact of the size of
+the training set on this trade-off**.
+
+Finally we will relate overfitting and underfitting to the concepts of
+statistical variance and bias.
+
+```{tableofcontents}ß
```
Comment (GaelVaroquaux) [inline jupyter-book/overfit/overfit_intro.md:21]: Naive question: is the "beta" at the end of this line desired?

I'm not quite sure what it does.

**Comment by GaelVaroquaux** [comment]
I notice that there are .ipynb files modified in this PR. Do we still need to commit these files? I thought that with the current build infrastructure they were no needed.

If they are needed, I am doing things wrong, as I am not updating them.

**Comment by GaelVaroquaux** [comment]
I'm in favor of merging this PR and iterating on top of this. Merging will make collaborative thinking on the MOOC easier.

File: jupyter-book/overfit/overfit_intro.md (hunk 1 above)
Comment (ogrisel) [reply jupyter-book/overfit/overfit_intro.md:21]: hehe, no it's a typo

**Comment by ogrisel** [comment]
I did not mean to commit those changes. I probably messed up something local git clone with another WIP branch when we changed the way to handle the notebooks. Let me clean this up.

File: jupyter-book/overfit/overfit_intro.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Overfitting and underfitting
+
+This chapter gives an intuitive introduction to the very **fundamental concepts** of
```
Comment (glemaitre) [inline jupyter-book/overfit/overfit_intro.md:3]: I think we should use "module" instead of "chapter" to be in line with the FUN denomination.

**Comment by ogrisel** [comment]
> I'm in favor of merging this PR and iterating on top of this. Merging will make collaborative thinking on the MOOC easier.

Let's merge. Feel free to edit directly and push to master. I opened the PR just to check that I did not break the jupyterbook rendering and it was worth it because I had not noticed that I had spurious changes in the notebooks in my branch.

**Comment by ogrisel** [comment]
Next: I will add quiz questions and then the progressive arrow throwing slides for the bias variance deck.

**Pull request merged**